reference/7.3/Microsoft.PowerShell.Core/About/about_Language_Modes.md

```diff
@@ -14,66 +14,170 @@ Explains language modes and their effect on PowerShell sessions.
 session.
 
-The language mode is actually a property of the session configuration (or
-"endpoint") that is used to create the session. All sessions that use a
-particular session configuration have the language mode of the session
+The language mode is a property of the session configuration (or "endpoint")
+that's used to create the session. All sessions that use a particular session
+configuration have the language mode of the session configuration.
+
+All PowerShell sessions have a language mode. Remote sessions are created using
+the session configurations on the remote computer. The language mode set in the
+session configuration determines the language mode of the session. To specify
+the session configuration of a PSSession, use the ConfigurationName parameter
+of cmdlets that create a session.
+
+## Finding the language mode of a session
+
+You can find the language mode of a `FullLanguage` or `ConstrainedLanguage`
+session by getting the value of the **LanguageMode** property of the session
+state.
+
+For example:
+
+```powershell
+$ExecutionContext.SessionState.LanguageMode
+ConstrainedLanguage
+```
+
+However, in sessions with `RestrictedLanguage` and `NoLanguage` modes, you
+can't use the dot method to get property values. Instead, the error message
+reveals the language mode.
+
+When you run the `$ExecutionContext.SessionState.LanguageMode` command in a
+`RestrictedLanguage` session, PowerShell returns the
+**PropertyReferenceNotSupportedInDataSection** and
+**VariableReferenceNotSupportedInDataSection** error messages.
+
+- **PropertyReferenceNotSupportedInDataSection**: Property references aren't
+  allowed in restricted language mode or a Data section.
+- **VariableReferenceNotSupportedInDataSection**: A variable that can't be
+  referenced in restricted language mode or a Data section is being
+  referenced.
+
+When you run the `$ExecutionContext.SessionState.LanguageMode` command in a
+NoLanguage session, PowerShell returns the **ScriptsNotAllowed** error message.
+
+- **ScriptsNotAllowed**: The syntax isn't supported by this runspace. This
+  might be because it's in no-language mode.
+
+## Finding the language mode of a session configuration
+
+When a session configuration is created using a session configuration file, the
+session configuration has a **LanguageMode** property. You can find the
+language mode by getting the value of the **LanguageMode** property.
+
+```powershell
+(Get-PSSessionConfiguration -Name Test).LanguageMode
+FullLanguage
+```
+
+On other session configurations, you can find the language mode indirectly by
+finding the language mode of a session that's created using the session
 configuration.
 
-All PowerShell sessions have a language mode, including PSSessions that you
-create by using the `New-PSSession` cmdlet, temporary sessions that use the
-ComputerName parameter, and the default sessions that appear when you start
-PowerShell.
+> [!NOTE]
```

Comment:
Actually, this is no longer true.  Starting with PS 7.3.0 you can start a PowerShell console session and configure that session by passing in a configuration file path.

```
-ConfigurationFile

    Specifies a session configuration (.pssc) file path. The configuration
    contained in the configuration file will be applied to the PowerShell
    session.

    Example: "pwsh -ConfigurationFile "C:\ProgramData\PowerShell\MyConfig.pssc"
```

Reply:
Thanks, I will update this.